### PR TITLE
tasks: Rename tests to tasks

### PR DESCRIPTION
**Pull request by Gundersanne**
- [x] rename the directory on bos cluster and add a symlink for old name
 - [x] move the images container over to cockpit-tasks-secrets and re-deploy it

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,8 +9,8 @@ docker-running:
 	systemctl start docker
 
 TAG := $(shell date --iso-8601)
-TEST_SECRETS := /var/lib/cockpit-tests/secrets
-TEST_CACHE := /var/cache/cockpit-tests
+TEST_SECRETS := /var/lib/cockpit-tasks/secrets
+TEST_CACHE := /var/cache/cockpit-tasks
```
Comment (martinpitt) [inline Makefile]: Let's rename the variables to `TASKS_*`, too. When deploying this container, we'll need to adjust this path on our bos cluster. We can do this now and add a tests → tasks symlink. (Creating a TODO item for that)

File: README.md [hunk 2]
```diff
@@ -6,5 +6,5 @@ The Cockpit Continuous Delivery Tools
  * learn: Machine learning for test results
  * release: container to run releases
  * sink: log sink used by testing infrastructure
- * tests: container used to select and run integration testing
+ * tasks: container used to select and run integration testing
```
Comment (martinpitt) [inline README.md]: This does way more these days; how about "run cockpit bots tasks like integration tests or image rebuilds"

**Comment by martinpitt** [comment]
The images/ container also uses the same tests/tasks volume and dir, so the renaming (and re-building/deploying) needs to happen there as well.

**Comment by martinpitt** [comment]
I renamed the directories and symlinked them on all our bos machines.

**Comment by martinpitt** [comment]
I built and deployed cockpit-tasks-secrets to both our clouds. Once you switched over and everything works, the old cockpit-tests-secrets should be removed.

As next steps, I suggest:

 - Uploading cockpit/tasks to dockerhub
 - Setting the replicationcontrollers to 0 on both clouds
 - deploying the new tasks yaml, and starting the pods
 - watch them for a while
   * if everything is good, remove the cockpit-tests related resources on both clouds
   * if something breaks, dial the tasks RCs to 0 and bring the tests RCs back up

**Comment by martinpitt** [comment]
Note that the images container also needs to be updated, see above. Therefore the "requested changes" remains for now.

File: images/.#cockpit-images.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+sanne@worky.17978:1542204001
```
Comment (martinpitt) [inline images/.#cockpit-images.yaml]: We don't want that bit :)

Comment (Gundersanne) [reply images/.#cockpit-images.yaml]: woops, guess I didn't setup my info/exclude on this one :)

**Comment by Gundersanne** [comment]
Rolled out on bos now too; only cockpit-11 needs a refresh of cockpit-images

File: images/install-service [hunk 4]
```diff
@@ -15,12 +15,12 @@ Requires=docker.service
...
 [Service]
-Environment="TEST_CACHE=$CACHE"
-Environment="TEST_SECRETS=$SECRETS"
+Environment="TASK_CACHE=$CACHE"
+Environment="TASK_SECRETS=$SECRETS"
 Restart=always
 RestartSec=60
 ExecStartPre=-/usr/bin/docker rm -f cockpit-images
-ExecStart=/bin/sh -xc "/usr/bin/docker run --name=cockpit-images --publish=8090:8080 --publish=8493:8443 --volume=\$TEST_SECRETS:/secrets:ro --volume=\$TEST_CACHE:/cache:rw cockpit/images"
+ExecStart=/bin/sh -xc "/usr/bin/docker run --name=cockpit-images --publish=8090:8080 --publish=8493:8443 --volume=\$TASK_SECRETS:/secrets:ro --volume=\$TASK_CACHE:/cache:rw cockpit/images"
```
Comment (martinpitt) [inline images/install-service]: The source ports are wrong, but I'll send a separate PR about that.

**Comment by martinpitt** [comment]
I removed the old cockpit-tests-secrets everywhere.

**Pull request merged**